### PR TITLE
Update create-a-custom-sensitive-information-type.md

### PR DESCRIPTION
**Pull request by maberzin**
Document 100 max EDM SIT evaluations (i.e., EDM trigger SIT detections). @chrfox  please review.

### CODE REVIEW

File: microsoft-365/compliance/create-a-custom-sensitive-information-type.md [hunk 1]
```diff
@@ -180,6 +180,8 @@ For a scanned item to satisfy rule criteria, the number of unique instances of a
 
 For example, if you want the rule to trigger a match when at least 500 unique instances of a SIT are found in a single item, set the **min** value to `500` and the **max** value to `Any`.
 
+Note: for EDM, we support up to 100 EDM SIT evaluations, thus policies should not be written with a Min or Max value greater than 100 for EDM SITs.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/create-a-custom-sensitive-information-type.md]:
```suggestion
> [!NOTE]
> We support up to 100 exact data match (EDM) evaluations, thus policies should not be written with a **min** or **max** instance count value greater than 100 for EDM SITs.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @maberzin, kindly commit the suggested change. Many thanks.

File: microsoft-365/compliance/create-a-custom-sensitive-information-type.md (hunk 1 above)
Comment (maberzin) [reply microsoft-365/compliance/create-a-custom-sensitive-information-type.md]: Overall looks fine but let's please add "instance count" - i.e., ...written with a **min** or **max** instance count value greater than 100 for EDM SITs

Comment (JohanFreelancer9) [reply microsoft-365/compliance/create-a-custom-sensitive-information-type.md]: I amended the suggested change accordingly. Could you please commit? Thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**